flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/AbstractFlinkService.java

```diff
@@ -338,6 +339,10 @@ protected void cancelJob(
                     } else if (ReconciliationUtils.isJobInTerminalState(deploymentStatus)) {
                         LOG.info(
                                 "Job is already in terminal state skipping cancel-with-savepoint operation.");
+                    } else if (deployment.getStatus().getReconciliationStatus().getState()
+                            == ReconciliationState.ROLLING_BACK) {
+                        LOG.info(
+                                "Job reconciliation status is in rolling back state. Job is expecting to not be in running or terminal state");
```

Comment:
We can't simply ignore errors here because the cluster may not have shut down. Please see my other comment, I dont think we should use savepoint cancel ever during rollbaack